packages/FormElement/src/FormElement.js

```diff
@@ -97,7 +97,6 @@ function FormElement(props) {
       {...moreProps}
     >
       <Label
-        hasInstructionsShowing={extractedChildren[subComponentDisplayNames.Instructions]}
```

Comment:
removing this as isn't needed, we can set the style on the instructions component itself https://github.com/acl-services/paprika/pull/766/files#diff-cfa086dd75b1cf0b6caf5b57c6f47887a53eb020345ca42c91687d628cd0389bR9